manifests/authzd_user.pp

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# keepalived
+#
+# @example
+#   include nebula::profile::authzd_user()
+define nebula::authzd_user(
+  String $home,
+  Hash $key,
+  String $gid) {
+
+  $key_file = "${home}/.ssh/authorized_keys"
+
+  file { $home:
+    ensure => 'directory',
+    mode   => '0755'
```

Comment:
I must have accidentally deleted this comment somehow. This resource could be omitted by adding `managehome => true` to the user resource. It'd accomplish the same thing, but it'd be more idiomatic puppet.

Reply:
Done.